performance_manager/lib/postgres_schema.py

```diff
@@ -108,13 +108,13 @@ class StaticRoutes(SqlBase):  # pylint: disable=too-few-public-methods
     pk_id = sa.Column(sa.Integer, primary_key=True)
     route_id = sa.Column(sa.String(60), nullable=False)
     agency_id = sa.Column(sa.SmallInteger, nullable=False)
-    route_short_name = sa.Column(sa.String(60), nullable=False)
-    route_long_name = sa.Column(sa.String(150), nullable=False)
-    route_desc = sa.Column(sa.String(40), nullable=False)
+    route_short_name = sa.Column(sa.String(60), nullable=True)
```

Comment:
why are all of these changing?

Reply:
This was an offshoot of the previous comment where values I thought didn't have any NULL records, actually did because empty strings were getting loaded.